.gitignore

```diff
@@ -1,8 +1,216 @@
+Skip to content
+Search or jump to…
+
+Pull requests
+Issues
+Marketplace
+Explore
+
+@neomonkeus
+34
+12253niftools/blender_nif_plugin
+ Code Issues 100 Pull requests 2 Projects 14 Security Insights Settings
+blender_nif_plugin/.gitignore
+@neomonkeus neomonkeus Stop jetbrains stuff annoying me, especially line length non-sense
+06d6c41 on 2 Jan
+@neomonkeus@amorilia@TagnumElite
+186 lines (147 sloc)  2.74 KB
```

Comment:
Uhh, what's this all related to?

Also, check [gitignore.io](https://www.gitignore.io/?templates=python,pycharm,visualstudiocode)

Reply:
eh... will look into it.
Using github gitignore